PayPal/src/main/java/com/braintreepayments/api/PayPalClient.java

```diff
@@ -113,12 +129,41 @@ public void onResult(@Nullable final Configuration configuration, @Nullable Exce
                     callback.onResult(manifestInvalidError);
                     return;
                 }
+                String clientId = configuration.getPayPalClientId();
+                if (clientId == null) {
+                    //In the future module, we should default this to our new callback function.
+                    //Is this the correct way to send an analytic?
+                    braintreeClient.sendAnalyticsEvent("paypal.native.client-id-null");
+                    callback.onResult(new BraintreeException("Client Id is null"));
+                    return;
+                }
```

Comment:
When I started the checkout with Client Id = null, i got an error. Is this the best way to handle it? As it is probably some problem that comes from the client.
Also, is that the correct way to send an analytic?

Reply:
I think it makes sense to callback an error. I'd guess the root issue is an improper BT <-> PayPal Merchant account linkage? Maybe we could provide a URL to `PayPalNative` docs in the error message.